assignments/E2.md

```diff
@@ -97,14 +102,13 @@ that it will create in `/proc`
 
     * The `/proc` file must be accessible at `/proc/kdlp`
 
-    * Update your makefile so it can build/clean,
-    load/unload, and test your kernel module
+    * Create a makefile in your `username/E2/` directory to build/clean and load/unload your module
 
     * The makefile's default and `module` targets must build a kernel object file named `kdlp.ko`
 
-    * The makefile's load and unload targets must load and unload your module respectively
```

Comment:
kinda want these fixes in their own commit, but if it is too much of a pain, you can leave it